client/src/InfoBase/App.js

```diff
@@ -132,114 +130,107 @@ export class App extends React.Component {
   render() {
     const { outage_msg, showSurvey } = this.state;
     return (
-      <div
-        tabIndex={-1}
-        id="app-focus-root"
-        className={`app-focus-root--${is_a11y_mode ? "a11y" : "standard"}`}
-      >
-        {is_a11y_mode && <NoIndex />}
```

Comment:
Lost this line during the rebase `{is_a11y_mode && <NoIndex />}` (and deleted the import in the following cleanup). Bring that back, thanks!

Reply:
Oh oops, my bad

Reply:
Just move it in to the `ErrorBoundary` this time, it never really needed to be outside of it.